libs/auth-api-lib/src/lib/user-notification/user-system-notification.config.ts

```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from '@island.is/nest/config'
+import { z } from 'zod'
+
+const schema = z.object({
+  basePath: z.string(),
+  scope: z.array(z.string()),
+})
+
+export const DelegationApiUserSystemNotificationConfig = defineConfig({
+  name: 'DelegationApiUserSystemNotificationConfig',
+  schema,
+  load(env) {
+    return {
+      basePath: env.required(
+        'USER_NOTIFICATION_CLIENT_URL',
+        'http://localhost:3333',
+      ),
+      scope: ['api_resource.scope'],
```

Comment:
You are not using the `scope` property? And we should never use the `api_resource.scope`. It is a legacy scope from the early days and we are trying to get rid of it from the codebase.